microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -776,6 +777,47 @@ See also:
 ---
 
 
+<!-- ------------------------------ -->
+#### Fluent Style Overlay Scrollbar
+<!-- todo: correct capitalization?  corp style is Sentence case for headings, unless product name.  exceptions: Release Notes, What's New. -->
+
+Support for the Fluent Style Overlay Scrollbar.
```

Comment:
describe Fluent Style Overlay Scrollbar for complete newbies in plain language that's relatable
@victorhuangwq @peiche-jessica 
eg hallway conversation w/ random person, summarizing the gist

Reply:
adaptive scrollbar design, adjusts to various devices and window sizes

Reply:
```
#### Fluent overlay scrollbars

Stylyzes scrollbars with Microsoft Fluent design and makes the scrollbars overlay 
over the web content.  This adaptive scrollbar design adjusts to various devices 
and window sizes.

To experiment with Fluent overlay scrollbars, in Microsoft Edge, go to `edge://flags` 
and then enter **Fluent overlay scrollbars**.

See also: 
* [WebView2 browser flags](./webview-features-flags.md) - `msEdgeFluentOverlayScrollbar`.
<!-- todo: article to link to? -->
```